src/bin/dual-iir.rs

```diff
@@ -228,10 +235,14 @@ const APP: () = {
             settings,
             signal_generator: [
                 SignalGenerator::new(
-                    settings.signal_generator[0].try_into().unwrap(),
+                    settings.signal_generator[0]
+                        .try_into_config(SAMPLE_TICKS_LOG2)
+                        .unwrap(),
                 ),
                 SignalGenerator::new(
-                    settings.signal_generator[1].try_into().unwrap(),
+                    settings.signal_generator[1]
+                        .try_into_config(SAMPLE_TICKS_LOG2)
```

Comment:
@jordens There's no other reason that `dual-iir` needs to restrict the sample rate to LOG2 values except for the signal generator, where we do an optimized division to calculate the FTW. Should we refactor the signal generator FTW calculation to support other tick sizes as well, or leave it for now?

Reply:
Leave it for now. That restriction can be removed later.